api/v1/words/get.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
May be we have to caching and all later for fetching the frequently fetched queries.
Hapi makes caching easier. :)